src/main/java/pro/belbix/ethparser/web3/contracts/ContractUtils.java

```diff
@@ -170,6 +171,12 @@ public static String getFarmAddress(String network) {
   }
 
   public static boolean isFullParsableLp(String address, String network) {
-    return FULL_PARSABLE_UNI_PAIRS.get(network).contains(address.toLowerCase());
+    return FULL_PARSABLE_UNI_PAIRS.get(network).containsKey(address.toLowerCase());
   }
-}
+
+  public static boolean isFullParsableLpAddressAndDate(String address, BigInteger date,
```

Comment:
why BigInteger? `int` enough

Reply:
because BigInteger ethLog.getBlockNumber() and I thought at the end to change the format